### ISSUE TITLE
ifconfig.io is down

### ISSUE THREAD

**Issue by tholu** [gstuartj/cf-ddns.sh#7]
```
external_WAN_query='https://ifconfig.io/ip'
```

The site is currently down, which breaks the script.

**Comment by tholu** [comment]
This works as a replacement: https://api.ipify.org/

**Comment by ConorIA** [comment]
ifconfig.io is working again.

**Comment by tholu** [comment]
As it seems unreliable, I would still switch it out.

**Comment by ConorIA** [comment]
Indeed, ifconfig.io also returns ipv6 addresses, which don't work with the script. Your recommendation fixes that issue.
